MetaMask/metamask-extension
# Support Incoming Transactions for Base

## Explanation



Currently when using MetaMask with Base you don't see incoming transactions (Receive entries) in the Activity tab.

This PR is a small quality of life improvement for Base + MetaMask users which updates the config to enable the incoming transactions. See this [PR](https://github.com/metaMask/metamask-extension/pull/14219) for when support was added to fetch incoming transactions via Etherscan API.

You can confirm the Etherscan endpoints here:
* [Testnet](https://api-goerli.basescan.org/api?module=account&action=txlist&address=0x7d5A90C2cd5567B9966E92c710b5E9936F400d74&tag=latest&page=1)
* [Mainnet](https://api.basescan.org/api?module=account&action=txlist&address=0x7d5A90C2cd5567B9966E92c710b5E9936F400d74&tag=latest&page=1)

## Screenshots/Screencaps



### Before

https://github.com/MetaMask/metamask-extension/assets/672580/16c93eb9-bb9d-4954-b7f8-8625fdedb2c1

### After

https://github.com/MetaMask/metamask-extension/assets/672580/68d01b8d-0585-4d10-ad10-66a6021c9b4e

## Manual Testing Steps

- Add the Base [testnet](https://chainlist.org/chain/84531) or [mainnet](https://chainlist.org/chain/8453) to Metamask
- Switch to that network
- Send funds from account 1 to account 2
- Switch to the account 2 activity tab and you should see a "Receive" entry for the funds you sent



## Pre-merge author checklist


## Pre-merge reviewer checklist


If further QA is required (e.g. new feature, complex testing steps, large refactor), add the `Extension QA Board` label.

In this case, a QA Engineer approval will be be required.
